pkgs/applications/science/biology/slicer3d/default.nix

```diff
@@ -0,0 +1,119 @@
+{
+  stdenv,
+  lib,
+  autoPatchelfHook,
+  makeDesktopItem,
+  copyDesktopItems,
+  glib,
+  fetchzip,
+  pulseaudio,
+  xorg,
+  zlib,
+  fontconfig,
+  libxkbcommon,
+  libGL,
+  libGLU,
+  freetype,
+  cups,
+  unixODBC,
+  libxcrypt-legacy,
+  hwloc,
+  alsa-lib,
+  postgresql,
+  nss,
+  nspr,
+}:
+
+let
+  pname = "Slicer3D";
+  version = "5.6.2";
+  stability = "release";
+  os = "linux";
+  old_hwloc = hwloc.overrideAttrs (_: {
+    version = "1.4.1";
+  });
+
+  desktopItem = makeDesktopItem {
+    name = "Slicer3D";
+    exec = "SlicerApp-real";
+    desktopName = "Slicer 3D";
+    genericName = "A 3D Slicer Program"; # edit later
+    categories = [ "Graphics" ]; # Edit later
+  };
+in
+stdenv.mkDerivation {
+  inherit pname version;
+  src =
+    let
+      name = "Slicer-${version}-linux-amd64.tar.gz";
+      url = "http://download.slicer.org/download?os=${os}&stability=${stability}&version=${version}";
+    in
+    fetchzip {
+      inherit name;
+      inherit url;
+      hash = "sha256-DmJS1yrwJBcAIRnVA8VdsO4u82E1MWX+uvRC+6dEXmM=";
+      extension = "tar.gz";
+    };
+
+  desktopItems = [
+    desktopItem
+  ];
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    copyDesktopItems
```

Comment:
In the installPhase you copy the desktopItem folder manually - is this hook actually used?